pkg/apis/openebs.io/scp/v1alpha1/storage-pool.go

```diff
@@ -29,8 +31,345 @@ import (
 type StoragePool struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
+	// Spec is the spec for a StoragePool resource
+	Spec StoragePoolSpec `json:"spec,omitempty"`
+	// Status is for handling status of StoragePool resource
+	Status StoragePoolStatus `json:"status,omitempty"`
 }
 
+// StoragePoolSpec is the spec for a StoragePool resource
+type StoragePoolSpec struct {
+	// StorageCohortName of the cohort the pool is a part of
+	StorageCohortName string `json:"storageCohortName,omitempty"`
```

Comment:
Id or name?

Reply:
Changed it to name as per the previous comments. cc @shovanmaity

Reply:
please follow https://github.com/openebs/device-localpv/pull/44/files#r703251609 for this.